tests/layer/test_hinsage.py

```diff
@@ -270,41 +270,6 @@ def test_hinsage_apply():
     assert actual == pytest.approx(expected)
 
 
-def test_hinsage_default_model():
```

Comment:
Should we be trying to keep/update this test?
```python
def test_hinsage_in_out_tensors():
    ...
    xin, xout = hs.in_out_tensors()
    ...
```

Reply:
I was too hasty to delete this one, I'll ad it back and rename it